tests/requirements.txt

```diff
@@ -1,14 +1,15 @@
+aiohttp==3.8.4
 base58==2.1.1
 black==24.10.0
 cryptography==43.0.1
 PyNaCl==1.5.0
 pytest==8.3.3
 pytest-asyncio==0.24.0
 pytest-timeout==2.3.1 
+pyYAML==6.0.1
 requests==2.28.2
 rlp==3.0.0
 ruff==0.7.0
```

Comment:
Are all of those still necessary?

Reply:
removed some unused libs. Got rid of all `request` lib usage.